src/style.css

```diff
@@ -1 +1,64 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  width: 100%;
+  padding: 27px 62px 26px;
+
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
```

Comment:
please write colors in lowercase like #e1e7ed, check all your scss file and update it